core/server/master/src/main/java/alluxio/master/file/meta/Inode.java

```diff
@@ -330,6 +332,20 @@ public T setMode(short mode) {
     return getThis();
   }
 
+  /**
+   * Sets ACL entries into the internal ACL.
+   * The entries will overwrite any existing correspondent entries in the internal ACL.
+   *
+   * @param entries the ACL entries
+   * @return the updated object
+   */
+  public T setAcl(List<AclEntry> entries) {
```

Comment:
This is still a merge, and not a set, right? Also, does this mean you can never remove an entry from the inode's acl, since this method only adds/merges an input list?

Reply:
yes, `entries` is merged into the internal ACL. This naming is consistent with the future shell command `setAcl` which sets (merges) ACL entries. There will be other methods for deleting entries from ACL in future PRs.